clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/loadbalancer/DeleteGoogleLoadBalancerAtomicOperation.groovy

```diff
@@ -92,64 +91,112 @@ class DeleteGoogleLoadBalancerAtomicOperation implements AtomicOperation<Void> {
 
     def timeoutSeconds = description.deleteOperationTimeoutSeconds
 
+    def numAttempts = 0
+    def operationSucceeded = false
+
     // Start deleting these objects. Wait between each delete operation for it to complete before continuing on to
     // delete its dependencies.
-    task.updateStatus BASE_PHASE, "Deleting forwarding rule $forwardingRuleName in $region..."
-    Operation deleteForwardingRuleOperation =
-        compute.forwardingRules().delete(project, region, forwardingRuleName).execute()
+    while (!operationSucceeded && numAttempts++ < MAX_NUM_DELETE_RETRIES) {
+      try {
+        task.updateStatus BASE_PHASE, "Deleting forwarding rule $forwardingRuleName in $region..."
+        Operation deleteForwardingRuleOperation =
+          compute.forwardingRules().delete(project, region, forwardingRuleName).execute()
 
-    googleOperationPoller.waitForRegionalOperation(compute, project, region, deleteForwardingRuleOperation.getName(),
-        timeoutSeconds, task, "forwarding rule " + forwardingRuleName, BASE_PHASE)
+        googleOperationPoller.waitForRegionalOperation(compute, project, region, deleteForwardingRuleOperation.getName(),
+          timeoutSeconds, task, "forwarding rule " + forwardingRuleName, BASE_PHASE)
 
-    def numAttempts = 0
-    def operationSucceeded = false
+        operationSucceeded = true
+      } catch (GoogleJsonResponseException e) {
+        if (e.details?.code == 404) {
+          if (numAttempts > 0) {
```

Comment:
Can this ever actually be `0` here? Maybe the incr neads to move down by the `sleep`?